hypershift-operator/controllers/infra.go

```diff
@@ -38,8 +38,7 @@ type InfrastructureStatus struct {
 func (s InfrastructureStatus) IsReady() bool {
 	return len(s.APIAddress) > 0 &&
 		len(s.OAuthAddress) > 0 &&
-		len(s.VPNAddress) > 0 &&
-		len(s.IgnitionProviderAddress) > 0
```

Comment:
we can drop the check, but we should also get the route hostname from the spec instead of status